database/csv.go

```diff
@@ -0,0 +1,86 @@
+package database
+
+import (
+	"context"
+	"encoding/csv"
+	"errors"
+	"log"
+	"os"
+	"strconv"
+
+	"github.com/TanZng/toh-api/models"
+)
+
+type CVSRepository struct {
+	file string
+}
+
+func NewCSVRepository(file string) (*CVSRepository, error) {
```

Comment:
If this is going to be a repository, you should separate it into its package called repository

Reply:
- [ ] CVSRepository to cvsRepository
- [ ] remove error unused
- [ ] move .csv to data folder and remove database folder
- [ ] move cvsRepository to repo folder